pystac/validation/__init__.py

```diff
@@ -164,7 +164,7 @@ class RegisteredValidator:
     def get_validator(cls) -> STACValidator:
         if cls._validator is None:
             try:
-                import jsonschema
+                import jsonschema as _  # noqa
```

Comment:
Why? Just to indicate it is unused in this module?

Reply:
Yeah, my [ruff](https://github.com/charliermarsh/ruff) linter was complaining about it, so this just makes it more explicit that it isn't being used.